src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --main-color: #000;
+  --link-color: #00acdc;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px ;
+}
+
+.logo {
+  display: block;
+  line-height: 0;
+}
+
+.logo__image {
+  display: block;
+  width: 100%;
+  object-fit: cover;
+  height: auto;
+}
+
+.nav {
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  text-transform: uppercase;
   margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
```

Comment:
👍 
This is needed for your approach of the `::after` element to work. Please spend some time explaining to yourself why

Hint: read carefully how the `absolute` element is positioned inside the `relative` container.